charmcraft/main.py

```diff
@@ -183,19 +183,19 @@ def _get_commands_info(self, commands_groups):
         for _cmd_group, _, _cmd_classes in commands_groups:
             for _cmd_class in _cmd_classes:
                 if _cmd_class.name in commands:
-                    _stored_class, _ = commands[_cmd_class.name]
+                    _stored_class = commands[_cmd_class.name]
                     raise RuntimeError(
                         "Multiple commands with same name: {} and {}".format(
                             _cmd_class.__name__, _stored_class.__name__
                         )
                     )
-                commands[_cmd_class.name] = (_cmd_class, _cmd_group)
+                commands[_cmd_class.name] = _cmd_class
```

Comment:
Here and a couple of lines above: stop having the group in the internal structure, as it's no longer used.